content/docs/user-guide/visualizing-plots.md

```diff
@@ -0,0 +1,334 @@
+# Visualizing Plots
```

Comment:
It's still low level to my mind. From reading this it's not clear (at least not immediately) how is different from just any plot lib. It reads like it's an alternative to that, while in our case the core differentiator is the approach - codification. DVC captures data into commits and then it can be extracted in different ways to do stuff (including plots). I think it's important to describe.

+ (or along the way) I would give a brief sense how it works - files are saved (can be written from a script in any format) + then DVC provides templates, tools etc to visualize this.

Reply:
> What do you think of limiting this PR to reorganizing the current content and leaving any content changes to a future PR?

@shcheklein Are you okay with this approach? Wanted to focus on where to put the existing content first.

Reply:
If we prioritize and do this quick - it makes sense. Otherwise I would be neutral about this PR.

Worst case - we move it now, next someone decides to move things other direction, and so on ... we'll be just moving things around. That's what we need to avoid at all cost.